File: TenMinsJava/day33.java
// print out current unix timestamp in seconds and milliseconds using built in functions if available

package TenMinsJava;

public class day33 {
    public static void main(String[] args) {
        long sec = System.currentTimeMillis()/1000;
        long millisec = System.currentTimeMillis();
        System.out.printf("Seconds: %d %nMiliseconds: %d", sec, millisec);
    }
}

File: TenMinsPy/day33.py
# print out current unix timestamp in seconds and milliseconds using built in functions if available

import time

class day33:
    def __init__(self):
        s = int(time.time())
        ms = int(time.time() * 1000)
        print(f"Seconds: {s} \nMiliseconds: {ms}")

day33()